### PR TITLE
EOS-22268: Manufacturing - Cron Job to change password for support user every 24 hours

### PR DESCRIPTION
**Pull request by sradhanand-pati**
Signed-off-by: sradhanandpati <sradhanand.pati@seagate.com>

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

File: srv/components/provisioner/scripts/support [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+from provisioner.utils import (
+    run_subprocess_cmd,
+    load_yaml
+)
+from provisioner.errors import SubprocessCmdError
+from cortx_setup.config import SUPPORT_USER_NAME
+
+#SUPPORT_CONFIG = Path('/var/data/support.yaml')
+SUPPORT_CONFIG = Path(f'/opt/seagate/users/{SUPPORT_USER_NAME}/support.yaml')
+#logging.basicConfig(
```
Comment (nitin-seagate) [inline srv/components/provisioner/scripts/support:39]: Lets enable logging for tracking purposes

**Review by nitin-seagate — APPROVED** [review]
Please attach test results

File: srv/components/provisioner/scripts/support (hunk 1 above)
Comment (sradhanand-pati) [reply srv/components/provisioner/scripts/support]: Sure. Done.

**Review by 83bhp — APPROVED** [review]

**Pull request merged**